langserver/util.go

```diff
@@ -31,3 +32,44 @@ func pathEqual(a, b string) bool {
 func IsVendorDir(dir string) bool {
 	return strings.HasPrefix(dir, "vendor/") || strings.Contains(dir, "/vendor/")
 }
+
+// isUri tells if s denotes an URI
+func isUri(s string) bool {
+	return strings.HasPrefix(s, "file://")
+}
+
+// pathToUri converts given path to file URI
+func pathToUri(path string) string {
+	prefix := "file://"
+	if !strings.HasPrefix(path, "/") {
+		// On Windows there should be triple slash
+		prefix += "/"
+	}
+	path = strings.Replace(path, "\\", "/", -1)
```

Comment:
> Can we instead rely on which operating we are running on to do this conversion?

It's more complicated then.. On Windows we may encounter slashes when working with VFS and backslashes (local FS)